src/Controller/Admin/IndexController.php

```diff
@@ -206,6 +206,8 @@ protected function buildPimcoreSettings(
             $adminEntrypointUrl = null;
         }
 
+        $requiredLangauges = $systemSettings['general']['required_languages'];
```

Comment:
Typo in variable name
```suggestion
        $requiredLanguages = $systemSettings['general']['required_languages'];
```